linkerd/app/src/env.rs

```diff
@@ -500,6 +520,51 @@ pub fn parse_config<S: Strings>(strings: &S) -> Result<super::Config, EnvError>
     })
 }
 
+fn oc_trace_labels(oc_file_path: Result<Option<String>, EnvError>) -> HashMap<String, String> {
+    let oc_labels = match oc_file_path {
+        Ok(path_option) => match path_option {
+            Some(path) => match fs::read_to_string(path.clone()) {
+                Ok(label_string) => convert_labels_to_map(label_string),
+                Err(err) => {
+                    warn!(
+                        "could not read OC trace labels file at path {}: {}",
+                        path, err
+                    );
+                    HashMap::<String, String>::new()
```

Comment:
Note that because the the return type of this function is `HashMap<String, String>`, the compiler will infer the type of this for you, and you could just write `HashMap::new()` in this function.